### PR TITLE
Upgrade zsh to 5.3.1

### PR DESCRIPTION
**Pull request by hboetes**
_Motivation:_ I wanted to upgrade zsh, as a first attempt to do something with the toolchain
_Linked issues:_ Optionally, add links to existing issues or other PR’s

### Checklist
- [ V ] Build rule `all-supported` completed successfully
- [ V ] Package upgrade completed successfully
- [ V ] New installation of package completed successfully

### CODE REVIEW

**Comment by KaraokeStu** [comment]
@Dr-Bean Do we usually increment the SPK revision, regardless of the main version, or is this update to the main version and reset of revision back to 1 going to be ok?

**Comment by hboetes** [comment]
Consistent with any other package management system I know the SPK_REV version is used for changes in the package while building the same upstream release, and should be reset to 1 when bumping the upstream package.

**Comment by cytec** [comment]
@KaraokeStu @hboetes `SPK_REV` is always +1 when the anything changes, thats more like an internal counter for us and will make sure that Package Center will show it as an update as well.

File: spk/tmux/Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 SPK_NAME = tmux
-SPK_VERS = 2.1
-SPK_REV = 2
+SPK_VERS = 2.3
+SPK_REV = 1
```
Comment (cytec) [inline spk/tmux/Makefile]: change to 3 as this always increments

File: spk/zsh/Makefile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 SPK_NAME = zsh
-SPK_VERS = 5.2
+SPK_VERS = 5.3.1
 SPK_REV = 3
```
Comment (cytec) [inline spk/zsh/Makefile]: same as above increment +1

**Comment by KaraokeStu** [comment]
Great thanks @cytec I'm pretty sure that was the case - @hboetes please could you increment the counter to "3" and then this PR is good to go!

**Comment by KaraokeStu** [comment]
Let me know when you've got those changes made and I'll approve the merge and get it building 👍

**Comment by KaraokeStu** [comment]
All done and published

**Pull request merged**